backend/RoomPlanner/room-planner/src/main/java/edu/roomplanner/service/impl/EncryptionServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package edu.roomplanner.service.impl;
+
+import edu.roomplanner.service.EncryptionService;
+import org.jasypt.util.text.BasicTextEncryptor;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class EncryptionServiceImpl implements EncryptionService {
+
+    private static char[] password;
```

Comment:
As your coleague suggested you should get the value of this variable using the @Value annotation.